victims/factory/daemonsets/daemonsets.go

```diff
@@ -44,20 +44,16 @@ func identifier(kubekind *appsv1.DaemonSet) (string, error) {
 
 // Read the mean-time-between-failures value defined by the DaemonSet
 // in the label defined by config.MtbfLabelKey
-func meanTimeBetweenFailures(kubekind *appsv1.DaemonSet) (int, error) {
+func meanTimeBetweenFailures(kubekind *appsv1.DaemonSet) (string, error) {
 	mtbf, ok := kubekind.Labels[config.MtbfLabelKey]
 	if !ok {
-		return -1, fmt.Errorf("%T %s does not have %s label", kubekind, kubekind.Name, config.MtbfLabelKey)
+		return "", fmt.Errorf("%T %s does not have %s label", kubekind, kubekind.Name, config.MtbfLabelKey)
 	}
 
-	mtbfInt, err := strconv.Atoi(mtbf)
+	_, err := time.ParseDuration(mtbf)
 	if err != nil {
-		return -1, err
+		return "", fmt.Errorf("error parsing mtbf %s: %v", mtbf, err)
```

Comment:
This makes it a breaking change. It would be good if we could still use values without a unit which would default to using days.